app/scripts/modules/amazon/src/loadBalancer/configure/application/TargetGroups.tsx

```diff
@@ -26,20 +26,17 @@ export interface ITargetGroupsState {
   oldTargetGroupCount: number;
 }
 
-class TargetGroupsImpl extends React.Component<
-  ITargetGroupsProps & IWizardPageProps & FormikProps<IAmazonApplicationLoadBalancerUpsertCommand>,
-  ITargetGroupsState
-> {
+class TargetGroupsImpl extends React.Component<ITargetGroupsProps, ITargetGroupsState> {
```

Comment:
I feel the component declarations are much easier to parse like this

Reply:
Uhhh... yes. Much easier. Awesome.